pkg/notes/document.go

```diff
@@ -31,7 +31,9 @@ func CreateDocument(notes ReleaseNoteList) (*Document, error) {
 		Uncategorized:  []string{},
 	}
 
-	for _, note := range notes {
+	for _, pr := range history {
+		note := notes[pr]
```

Comment:
I can't think of any case where something in the `history` wouldn't also be in `notes` (that would require a previously-merged PR disappearing) but it might be good/safe to trap that edge case in the future.